airbyte-integrations/connectors/source-tiktok-marketing/acceptance-test-config.yml

```diff
@@ -28,22 +28,22 @@ tests:
   discovery:
     - config_path: "secrets/prod_config.json"
       backward_compatibility_tests_config:
-        disable_for_version: "0.1.17"
+        disable_for_version: "1.0.0"
```

Comment:
Removing streams are considered a breaking change, however we need to do this because the streams aren't usable any more. We will communicate this to the customers attempting to use these non-working streams to reset or disable them.